VOM_Fortran/VOM-code/transpmodel.f90

```diff
@@ -1528,6 +1540,90 @@ end subroutine vom_write_dayyear
 !+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
 !+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
 
+     subroutine vom_save_dayyear (output)
+      use vom_vegwat_mod
+      implicit none
+
+      CHARACTER(60) :: dailyformat
+      CHARACTER(3)  :: str
+      REAL*8, dimension(:), allocatable        :: tmp
+      REAL*8, DIMENSION(21, c_maxday ), intent(inout)        :: output
+
+!     * internal write to convert from number to string
+      write(str,'(I3)') wlayer_
+!     * includes a column for each sublayer
+      dailyformat = '(I6,I6,I4,I7,'//str//'E14.6)'
+
+    if(nday .eq. 1) then
+
+    !allocate(output(21, c_maxday))
+
+       output(1, nday) = etmt_d
+
+    else
+
+       output(1, nday) = etmt_d
+
+    end if 
+
+    !write to file
+    if(nday .eq. c_maxday) then
+
+
+
+       write(kfile_etmt,*) output(1,:)
+
+
+
+    end if 
+
+
+!      write(kfile_resultsdaily,'(I6,I7,I7,I7,I7,25E15.5)')             &
+!     &  fyear(nday), fmonth(nday), fday(nday), nday, nhour-1,          &
+!     &  rain_d(nday), tairmax_d(nday), tairmin_d(nday), par_d(nday),   &
+!     &  vd_d / 24.d0, esoil_d, jmax25t_d(2), jmax25g_d(2),             &
+!     &  o_pct + pcg_d(2), rlt_d + rlg_d, lambdat_d, lambdag_d,         &
+!     &  rrt_d * 3600.d0 * 24.d0, rrg_d * 3600.d0 * 24.d0, asst_d(2),   &
+!     &  assg_d(2,2), SUM(su__(1:wlayer_)) / wlayer_, zw_, wsnew,       &
+!     &  spgfcf_d, infx_d, etmt_d, etmg_d, su__(1), topt_
+
+!        write(kfile_rsurfdaily,dailyformat) fyear(nday), fmonth(nday), &
+!     &    fday(nday), nday, rsurft_(1:wlayer_)
+
+!      if (fyear(nday) .ne. nyear) then
+!       * for calculation of vd_y a -1 is added to nday for using dayyear of correct year
+!        write(kfile_resultsyearly,'(i6,18e16.6)') nyear, rain_y,       &
+!     &    par_y, srad_y, vd_y / (dayyear(nday-1)), esoil_y, etm_y,     &
+!     &    etmg_y, assg_y, rlg_y, rrg_y, cpccg_y, tcg_y,                &
+!     &    etmt_y, asst_y, rlt_y, rrt_y, cpcct_y, tct_y
+!      endif
+
+!     * WRITING THE ACCUMULATED DATA FROM THE LAST YEAR TO FILE:
+
+!      if (nday .eq. c_maxday) then
+!       * call subroutine there to get yearly data for the output
+!        call vom_add_yearly()
+!        write(kfile_resultsyearly,'(i6,18e16.6)') nyear, rain_y,       &
+!     &    par_y, srad_y, vd_y / (dayyear(nday)), esoil_y, etm_y,       &
+!     &    etmg_y, assg_y, rlg_y, rrg_y, cpccg_y, tcg_y,                &
+!     &    etmt_y, asst_y, rlt_y, rrt_y, cpcct_y, tct_y
+!      endif
+
```

Comment:
If the commented out code is not needed any more, it could be removed.

Reply:
Removed commented code